service/matching/version_sets_test.go

```diff
@@ -41,7 +41,7 @@ import (
 
 func mkNewSet(id string, clock clockpb.HybridLogicalClock) *persistencepb.CompatibleVersionSet {
 	return &persistencepb.CompatibleVersionSet{
-		SetIds:                 []string{id},
+		SetIds:                 []string{hashBuildID(id)},
```

Comment:
no need to hash in the tests if it makes them more readable. but also fine to do it for consistency.

Reply:
It just simplifies the the test logic because the implementation hashes the build IDs.